zstackctl/zstackctl/ctl.py

```
@@ -12215,18 +12219,59 @@ def _wait_for_morph(self, url, timeout=600):
+    def _wait_for_license_server(self, urls, timeout=120):
+        info_and_debug("Waiting for %s to become available at: %s" % (self.service_name(), ", ".join(urls)))
+        begin = time.time()
+
+        while time.time() - begin < timeout:
+            for url in urls:
+                try:
+                    result = shell("curl -k -s -o /dev/null -w '%%{http_code}' %s" % url)
+                    if result.strip() == "200":
+                        info_and_debug("%s is ready at %s." % (self.service_name(), url))
+                        return url
+                except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)